apps/src/templates/projects/PublicGallery.jsx

```diff
@@ -0,0 +1,10 @@
+import React, {Component} from 'react';
+
+class PublicGallery extends Component {
+  render() {
+    return (
+      <h1>Public Gallery</h1>
```

Comment:
Yes. there are going to be a fair number of translations specific to `/projects`. Thoughts on whether it makes sense to have a new locale bundle for projects, or best just to put this in common?